extension.css

```diff
@@ -21,25 +22,24 @@ body * {
   text-align: start;
 }
 
-.browser-style {
+.browser-style,
```

Comment:
I think, if we're extracting this out to a style-sheet of its own, we shouldn't need to also force people to add a `class="browser-style"` to their html…

Reply:
Good point. Probably if we would opt for a stand-alone stylesheet we wouldn't need to have those classes but the purpose of this project is to photonize `extension.css` which currently uses those classes https://searchfox.org/mozilla-central/source/browser/components/extensions/extension.css#32. What I tried to do is not to change the class naming too much this is why I kept those classes. The end goal once we are happy with the visual styles is to create a patch on bugzilla to update those styles. Or are you suggesting to remove those classes altro from the `extension.css` hosted on mozilla-central?

Reply:
I was also talking with the webextension team and most of add-ons author do not update their modules (scenario of an author who installed the photon styles package) so it would be more convenient for us, for the time being, to keep those styles on mozilla-central.

Reply:
Yes, I'm suggesting that we don't force add-on authors who ask for browser-style (which is the case this css file is used) to add a bunch of extra classes to their pages. It might be a decent idea to run that by the webextension team first, though, and see what they say.

Reply:
I just talked with kmag (who's visiting Toronto!) and he agrees that we should remove the `.browser-style` from the checks. If people complain, we could add a `:not(.plain)` to let them disable it on a case-by-case basis, but we shouldn't pre-emptively do that. 😉